src/CodeGen_WebAssembly.cpp

```diff
@@ -110,13 +125,114 @@ void CodeGen_WebAssembly::init_module() {
     }
 }
 
-string CodeGen_WebAssembly::mcpu() const {
+void CodeGen_WebAssembly::codegen_vector_reduce(const VectorReduce *op, const Expr &init) {
+#if LLVM_VERSION >= 130
+    struct Pattern {
+        VectorReduce::Operator reduce_op;
+        int factor;
+        Expr pattern;
+        const char *intrin;
+        Type narrow_type;
+        uint32_t flags;
```

Comment:
You don't need this either, no patterns use any flags.

Reply:
See above, ditto.